### PR TITLE
Fix st2-auth-ldap circleci, remove deb/rpm packaging

### PR DESCRIPTION
**Pull request by amanda11**
Fix st2-auth-ldap CircleCI after it is included in ST2 and not a separate package.

Updated to run flake8 and pylint on circleCI

Added CircleCI badge to README

Renamed unit-tests that had same name.

Fixed issue with test-dependencies between pylint 2.6.0 and isort

Closes https://github.com/StackStorm/st2-auth-ldap/pull/74

### CODE REVIEW

**Comment by armab** [comment]
Just updated the Github settings to reflect your CI changes.
Please also remove the TravisCI config from this repo, - it's unnecessary here and will stop reporting the status.

**Comment by armab** [comment]
Would be also nice to include a new CircleCI status badge in the README.md.

**Comment by amanda11** [comment]
> Would be also nice to include a new CircleCI status badge in the README.md.

I think I have - well I copied some code from st2-rbac-backend and modified it - so I'm hoping that will have done that.

Thanks for update of repo. Travis.yml removed.

I'll change this to ready for review.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,156 +1,48 @@
 # Setup in CircleCI account the following ENV variables:
 # IS_PRODUCTION (default: 0)
-# IS_ENTERPRISE (default: 0)
 # PACKAGECLOUD_ORGANIZATION (default: stackstorm)
 # PACKAGECLOUD_TOKEN
```
Comment (armab) [inline .circleci/config.yml:4]: These lines could be removed as well.

Previously used for deploying the deb/rpm to PackageCloud, obsolete now with the packages removal.

**Review by armab — APPROVED** [review]
Great work on making the LDAP codebase actual and working!

Thanks a lot :+1:

**Pull request merged**